stable/pomerium/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: pomerium
-version: 4.1.1
+version: 4.1.2
 appVersion: 0.5.1
```

Comment:
shouldn't we take this opportunity to up the version here too @travisgroth?

Reply:
Since this is coming in as a third party patch, I'll do that as a follow up.